samples/subsys/input/input_dump/src/main.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/input/input.h>
+
+static void input_cb(struct input_event *evt, bool sync)
+{
+	printk("input event: dev=%-16s %3s type=%2x code=%3d value=%d\n",
+	       evt->dev->name,
+	       sync ? "SYN" : "",
+	       evt->type,
+	       evt->code,
+	       evt->value);
+}
+INPUT_LISTENER_CB_DEFINE(NULL, input_cb);
```

Comment:
Acheable with zbus.

Reply:
Let's followup with more features such as zbus after this is merged.